### ISSUE TITLE
Icons not showing on start

### ISSUE THREAD

**Issue by benyanke** [serge-rider/dbeaver#2393]
![image](https://user-images.githubusercontent.com/4274911/31837615-1c78df48-b59f-11e7-9c0c-4225f64f04eb.png)

On start, the icons do not show. Version 4.2.3 CE.

**Comment by vs0uz4** [comment]
At the beginning, icons are not displayed in version 4.2.4 CE as well.

**Comment by vs0uz4** [comment]
Interestingly, if you open dbeaver with a table from some database already open, the icons appear normally
![captura de tela de 2017-11-12 15-46-41](https://user-images.githubusercontent.com/2080547/32701672-06bd4a3e-c7c1-11e7-9656-9eb082d67035.png)

**Comment by serge-rider** [comment]
Is this still happens? 
(Maybe it was resolved after migration to Eclipse 4.7)

**Comment by benyanke** [comment]
Yes, it does - don't have much time to debug, crunch time at work. I can usually get it to stop by resizing the window a few times, so not a critical bug, but it is indeed still there.

Once this project is behind me, I'll take a closer look and help you debug further.
